src/agent/protocols/hack/update-generated-proto.sh

```diff
@@ -1,142 +1,188 @@
 #!/bin/bash
 
-# //
-# // Copyright 2020 Ant Financial
-# //
-# // SPDX-License-Identifier: Apache-2.0
-# //
+# Copyright 2020 Ant Financial
```

Comment:
might be you should update the `Copyright` too :smile:

Reply:
Done ;)

Reply:
thanks